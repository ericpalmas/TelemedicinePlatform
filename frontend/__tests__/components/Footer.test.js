import React from 'react'
import renderer from 'react-test-renderer'
import Footer from '../../src/components/Footer'
import { render, screen } from '@testing-library/react'
import 'core-js/stable'
import 'regenerator-runtime/runtime'

describe('Testing footer component', () => {
  test('footer renders correctly', () => {
    const tree = renderer.create(<Footer />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  //   test('footer text renders correctly', async () => {
  //     const { getByTestId } = render(<Footer />)
  //     const { getByText } = within(getByTestId('footerText'))
  //     expect(getByText('Copyright Â© MyWebSite')).toBeInTheDocument()
  //   })
})
