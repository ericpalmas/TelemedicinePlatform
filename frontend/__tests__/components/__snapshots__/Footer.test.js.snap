// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing footer component footer renders correctly 1`] = `
<footer>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="text-center py-3 col"
        id="footerText"
      >
        Copyright Â© MyWebSite
      </div>
    </div>
  </div>
</footer>
`;
